ui/desktop/electron-app/src/helpers/spawn-promise.js

```diff
@@ -78,6 +78,12 @@ module.exports = {
    * @returns {{stdout: string | undefined, stderr: string | undefined}}   */
   spawnSync(args, envVars = {}) {
     const childProcess = spawnSync(path(), args, {
+      // Some of our outputs (namely client daemon searching) can be very large.
+      // This an undocumented hack to allow for an unlimited buffer size which
+      // could change at any time. If it does, we should just set an arbitrarily
+      // large buffer size or switch to spawn and stream the output and handle
+      // killing the process ourselves.
+      maxBuffer: undefined,
```

Comment:
Do people prefer I just set an arbitrarily large buffer size instead rather than use an undocumented behavior? If so, what should I set it to?

Reply:
I think that for now, this hack is good but perhaps we can revisit this topic during techtime?